packages/syft/src/syft/core/tensor/passthrough.py

```diff
@@ -55,13 +46,65 @@ def __len__(self) -> int:
         return len(self.child)
 
     @property
-    def shape(self):
-        return tuple(self.child.shape)
+    def shape(self) -> Union[TypeTuple[Any, ...], List[Any]]:
```

Comment:
Perhaps just use TypeTuple[int] and cast to tell mypy thats what it is.

Reply:
done